easycv/transforms/color.py

```diff
@@ -88,3 +88,15 @@ def process(self, image, **kwargs):
         return cv2.stylization(
             image, sigma_s=kwargs["smoothing"], sigma_r=kwargs["region_size"]
         )
+
+
+class PhotoSketch(Transform):
+    """
+    PhotoSketch is a transform that creates a black and white pencil-like drawing.
+    """
+
+    def process(self, image, **kwargs):
+        img_gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+        img_blur = cv2.GaussianBlur(img_gray, (21, 21), 0, 0)
+        img_blend = cv2.divide(img_gray, img_blur, scale=256)
```

Comment:
Don't implement this by hand. Use Opencv builtin functions. Also, implement 2 methods color and grayscale pencil drawing. Check [this post.](https://stackoverflow.com/questions/42073941/convert-image-to-pencil-drawing-not-pencil-sketch-using-imagemagick) Ask Neutel for help with the methods.